.github/workflows/manual-deploy-kafka-aiven.yaml

```diff
@@ -0,0 +1,57 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+
+jobs:
+  setup-matrix:
```

Comment:
Her skjedde det mye.. men man kan velge dev eller prod her?

Reply:
nei. den går til både dev og prod. det matrix-greiene er for å støtte at vi legger til flere topics.